### ISSUE TITLE
Docs not in sync with pypi release 0.2.1 (latest); version bump on pypi? Tracking tasks / changes?

### ISSUE THREAD

**Issue by nickdirienzo** [theelous3/asks#22]
Hey @theelous3 et al. Really cool package—I found out about it from one of the trio PRs. I'm excited to play around with it.

Unfortunately, as I was trying to get started, I ran into some trouble with invoking `asks.init` per the [docs](https://asks.readthedocs.io/en/latest/#importing-asks). However, it seems the version on pypi and the docs are not in sync right now.

I saw that some changes went in a couple hours ago via this [commit](https://github.com/theelous3/asks/commit/ce8b6d26c119806db29424b0dfa5779167928cc5). What's the timeline to get the trio and curio support released? Is a released blocked by #21? 

I'm also wondering if there's a way I can keep a pulse on (or help with) changes coming down the pike. I really love what's going on here and would like to use it! :)

Thanks y'all!

**Comment by theelous3** [comment]
Hi there. Thank you!

I'm going to bump it on pypi within the next few hours. In the mean time you can `pip install git+https://github.com/theelous3/asks` to play around with it.

Release is not blocked by pending PRs, and I believe is ready to roll.

> I'm also wondering if there's a way I can keep a pulse on (or help with) changes coming down the pike.

Contribs are absolutely welcome. I tend to keep that stuff noted [here](http://asks.readthedocs.io/en/latest/#the-future) under "The Future", which I'm about to update actually. The next step aside from minor house cleaning is to use stdlib cookies.

This whole project when it began was dependantless. I handled all of the http and cookies etc. in asks as an educational exercise. Terrible for long term use however, so I switched the http stuff out to h11, and now I want to switch the cookie stuff out to CookieLib.

Thanks again for the high praise. It is very much appreciated. Feel free to drop in to the #8banana irc channel on freenode if you have any thoughts / suggestions / just want to shoot the shit regarding asks.

**Comment by nickdirienzo** [comment]
Thanks for the fast and detailed reply, @theelous3! Looking forward to seeing those changes published.

For sure. I figured I could install via git, but wanted to try stable first.

This is an awesome educational project. Nice work so far! I'm familiar with `asyncio`'s ecosystem; trying to wrap my head around `curio` and `trio` as well as their associated libraries.

That "The Future" section seems like a good place for now. I filed the cookie refactor under #23, hope you don't mind! The changelog would be great too; I've seen projects keep that at the top-level of the project under a `CHANGELOG` or `CHANGES`, e.g. Flask. But whatever you think is best. I'm a noob around here.

Neat! I'll follow along in #8banana to hang out with y'all, ha. Thanks again!

**Comment by nickdirienzo** [comment]
Uh oh. I'm seeing a problem with what I'm using from `master`. Do you want me to file this as a separate issue?

Originally I tried running with `trio-0.1.0`, but that failed with:
```
Traceback (most recent call last):
  File "app.py", line 3, in <module>
    asks.init('trio')
  File "/Users/ndirienzo/.virtualenvs/dnd-oncall/lib/python3.6/site-packages/asks/__init__.py", line 72, in init
    from ._event_loop_wrappers import (trio_open_connection,
  File "/Users/ndirienzo/.virtualenvs/dnd-oncall/lib/python3.6/site-packages/asks/_event_loop_wrappers.py", line 11, in <module>
    from trio import open_tcp_stream, open_ssl_over_tcp_stream
ImportError: cannot import name 'open_tcp_stream'
```

Using `trio-0.1.0+dev`, works but only over HTTP on Mac OS X 10.12.5. I think this is because the SSL certs on the OS moved around (I've ran into this before). I'll have to figure out how to set cert path for trio, it seems.

**Comment by theelous3** [comment]
Roger dodger. I knew there was something I was forgetting. trio support requires using the trio dev branch. As far as I can see it's reflectant of the future stable api in terms of high level socket and file io. The current trio on pypi doesn't at all have the features asks requires. Going to throw a nice bit note about that in the readme!
